_pages/contact.md

```diff
@@ -1,5 +1,8 @@
 ---
 title: "Contact Us"
+
+layout: footerimage
+footerimage: "contact.JPG"
```

Comment:
Hah, yup, forgot to handle that (it's been downloaded direct from my Google Photos which is a good job as straight off the camera they are closer to 12mb) - I'll sort that. Any preference for format?